src/docker-it/Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:14.04
+
+RUN apt-get install -y ssh && apt-get update && mkdir /var/run/sshd && \
```

Comment:
@vkuchyn `apt-get update` must come before `apt-get install` always!

Also we're still missing the version on `ssh`.

Please change this to:

`RUN apt-get update && apt-get install -y ssh=1:6.6p1-2ubuntu2.6`